### PR TITLE
Update block-events.md

### PR DESCRIPTION
**Pull request by VactricaKing**
I have discovered a better way to place the blocks using "set_block" and "set_block_at_pos" it seems almost nobody knows it, so I contribute something that in my opinion will greatly improve how to make a double block.

### CODE REVIEW

**Comment by VactricaKing** [comment]
@SmokeyStack Why is it not accepted yet? Is there a problem with my contribution?

File: docs/blocks/block-events.md [hunk 1]
```diff
@@ -283,7 +284,13 @@ Removes the current block and replaces it with the defined block in the same pos
 ```json
 "wiki:place_block": {
   "set_block": {
-      "block_type": "minecraft:grass"
```
Comment (SmokeyStack) [inline docs/blocks/block-events.md]: Please make this a separate codeblock and subheader. That way we can have a normal `set_block` and one with block states

File: docs/blocks/block-events.md [hunk 2]
```diff
@@ -295,10 +302,15 @@ Sets a block at a specified position relative to the block.
 <CodeHeader>minecraft:block > events</CodeHeader>
 
 ```json
-"wiki:generate_stone_above": {
```
Comment (SmokeyStack) [inline docs/blocks/block-events.md]: Please make this a separate codeblock and subheader. That way we can have a normal set_block and one with block states

**Review by SmokeyStack — CHANGES_REQUESTED** [review]

**Comment by VactricaKing** [comment]
@SmokeyStack Done!

**Review by SmokeyStack — APPROVED** [review]

**Pull request merged**